Code/Source/RobotImporter/URDF/RobotImporterInputInterface.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/string/string.h>
+
+namespace ROS2
+{
+    class RobotImporterInputInterface
```

Comment:
I changed the name to `RobotImporterUserInteractions` first but in the end, I decided to remove this entirely and simply pass the widget for whatever needs the user interactions (RobotImporter and URDFPrefabMaker at the moment)

> s there any other sensible way of implementing this interface than what is done by the Widget? 

It could be implemented in a test environment as a user mockup but I am not sure whether such tests ever emerge. When it would be useful, this interface might be brought back